src/views/transaction-details/transaction-details.view.tsx

```diff
@@ -136,9 +199,29 @@ const TransactionDetails: FC = () => {
           </Typography>
           <Chain chain={destinationNetwork} className={classes.alignRow} />
         </div>
+        {step1Fee && (
+          <div className={classes.row}>
+            <Typography type="body2" className={classes.alignRow}>
+              Step 1 Fee ({getChainName(transaction.data.bridge.networkId)})
+            </Typography>
+            <Typography type="body1" className={classes.alignRow}>
+              {step1Fee} {env.tokens.ETH.symbol}
+            </Typography>
+          </div>
+        )}
+        {step2Fee && (
+          <div className={classes.row}>
+            <Typography type="body2" className={classes.alignRow}>
+              Step 2 Fee ({getChainName(transaction.data.bridge.destinationNetwork)})
+            </Typography>
+            <Typography type="body1" className={classes.alignRow}>
+              {step2Fee} {env.tokens.ETH.symbol}
+            </Typography>
+          </div>
+        )}
         <div className={classes.row}>
           <Typography type="body2" className={classes.alignRow}>
-            Step 1/2
+            Step 1 Transaction
```

Comment:
I miss the word `Track` here as it was in the original designs from Chami. I would label this as: `Track step X transaction`

Reply:
Done